tests/bugs/issue681.js

```diff
@@ -0,0 +1,46 @@
+const Filer = require('../../src');
```

Comment:
You can also use 
``` JS
'use strict';
```
know more about strict here > https://www.geeksforgeeks.org/strict-mode-javascript/